esp-hal-common/src/clock/clocks_ll/esp32c6.rs

```diff
@@ -41,7 +41,7 @@ const MODEM_LPCON_CLK_I2C_MST_FO: u32 = 1 << 2;
 const MODEM_LPCON_I2C_MST_CLK_CONF_REG: u32 = DR_REG_MODEM_LPCON_BASE + 0x10;
 const MODEM_LPCON_CLK_I2C_MST_SEL_160M: u32 = 1 << 0;
 
-pub(crate) fn esp32c6_rtc_bbpll_configure(xtal_freq: XtalClock, pll_freq: PllClock) {
+pub(crate) fn esp32c6_rtc_bbpll_configure(_xtal_freq: XtalClock, _pll_freq: PllClock) {
     unsafe {
         // enable i2c mst clk by force on temporarily
         (MODEM_LPCON_CLK_CONF_FORCE_ON_REG as *mut u32).write_volatile(
```

Comment:
Are these not available in the PAC?

Reply:
We're still missing register description files for a number of peirpherals.